api/migration/read

```diff
@@ -75,19 +76,59 @@ def get_webmail_info():
     }
 
 
-try:
+def get_mail_info():
+    return {
+        "id": "nethserver-mail",
+        "name": "Email"
+    }
+
+
+def get_webtop_info():
+    config = get_config("webtop")
+
+    return {
+        "id": "nethserver-webtop5",
+        "name": "WebTop",
+        "config": config
+    }
+
+
+def list_apps():
     apps = []
     apps.append(get_account_provider_info())
     apps.append(get_nextcloud_info())
     apps.append(get_mattermost_info())
+    apps.append(get_mail_info())
     apps.append(get_webmail_info())
+    apps.append(get_webtop_info())
+    return apps
+
+
+def get_cluster_status():
+    bash_command = "/usr/bin/ns8-action cluster get-cluster-status null"
+    process = subprocess.Popen(bash_command.split(), stdout=subprocess.PIPE)
+    output, error = process.communicate()
+    return simplejson.loads(output)
 
-    for app in apps:
-        status = getMigrationStatus(app["id"])
-        app["status"] = status
 
-    output = simplejson.dumps({'migration': apps})
-    print(output)
+try:
+    line = fileinput.input()[0].rstrip()
+    input_json = simplejson.loads(line)
```

Comment:
To read JSON data from stdin I'd rather use the `load()` function

See https://simplejson.readthedocs.io/en/latest/#simplejson.load

```suggestion
    input_json = simplejson.load(sys.stdin)
```